steamed_hams.py

```diff
@@ -170,10 +172,10 @@ def steamed_hams_parser():
 
 for mod in modules:
     opts = python_opts.format(module=mod)
-    modules[mod] = "python3.9 {opts} {module}/{module}.py".format(opts=opts, module=mod)
+    modules[mod] = "python{version} {opts} {module}/{module}.py".format(opts=opts, module=mod, version=PYTHON_VERSION)
```

Comment:
```suggestion
    modules[mod] = "python{version} {opts} {module}/{module}.py".format(version=PYTHON_VERSION, opts=opts, module=mod)
```

I just... couldn't handle it being out of order :p